.gitignore

```diff
@@ -6,4 +6,8 @@ coverage
 
 dist
 
-.vscode
+#Cucumber tests
+/behaviours/reports/cucumber
+/behaviours/reports/templateCoverage
+/behaviours/reports/fixtures/temp
+/behaviours/env/default/tests.secrets.js
```

Comment:
If the goal is to just make sure that `fullFixture.json` is not excluded. Convey that intent in the syntax.

````
behaviours/reports/*
!/behaviours/reports/fixtures/fullFixture.json
````